src/middlewared/middlewared/plugins/pool.py

```diff
@@ -466,6 +466,7 @@ def pool_extend(self, pool):
         Str('name', required=True),
         Bool('encryption', default=False),
         Str('deduplication', enum=[None, 'ON', 'VERIFY', 'OFF'], default=None, null=True),
+        Int('special_small_block_size', validators=[Range(min=512)]),
```

Comment:
Is this a create only option?

Reply:
@sonicaj if it isnt, I dont see why we should put this option in pool create?

Why is this any more special that all others properties?

Reply:
I thought it might be beneficial for the users to specify this for the root dataset in one step of pool creation like we do for `deduplication`, I'll have it removed.